### DISCUSSION TITLE
Capability to link between recipes

### DISCUSSION THREAD

**Discussion by diegoroccia** [mealie-recipes/mealie#4629] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

As part of my personal recipes database, which I am porting to Mealie, I have multiple "base recipes" that serve as building blocks for other recipes (e.g., Bolognese sauce, Besciamelle). In Mealie, I need to repeat the same steps in each recipe (e.g., Lasagna Bolognese), or I won't get all the ingredients added to the shopping list. However, this creates a downside: I end up with the same recipe duplicated multiple times, and when I improve, for example, my Besciamelle recipe, I have to manually update all its instances in every other recipe.

### Please provide a concise description of the feature that would resolve your issue.

One solution would be to add the ability to link recipes, for example, by introducing a step type like "link another recipe." This step would simply render the link, but when adding ingredients to the shopping list, Mealie could loop through the linked recipes and include all their ingredients.

### Please consider and list out some caveats or tradeoffs made in your design decision

- visibility of the steps of the base recipe would be a bit worse, and the user would need to open multiple tabs to keep track of it.
-How to handle tags from the base recipe? should they be also considered tags for the "main" recipe?

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by diegoroccia** [comment]
> If this is accepted I'm willing to submit a PR to provide this feature

I didn't check this because I am not a frontend developer. If there's something in the python backend, I can help with that for sure

**Comment by Kuchenpirat** [comment]
There is already an open discussion around that.
See #1761
I'm gonna close this as duplicate.

If you want to add any ideas or request you are more than welcome to contribute to the discussion over there.

**Comment by diegoroccia** [comment]
Thank you, and sorry, I looked in the discussions but didn't notices it. I will check the discussion out there
